README.md

```diff
@@ -101,6 +101,27 @@ A [sample script](https://github.com/PandorasFox/i3lock-color/blob/master/lock.s
 On OpenBSD the `i3lock` binary needs to be setgid `auth` to call the
 authentication helpers, e.g. `/usr/libexec/auth/login_passwd`.
 
+Building i3lock
+---------------
+We recommend you use the provided package from your distribution. Do not build
```

Comment:
Remove this paragraph, as i3lock-color doesn't generally have packages available from distros.